tests/moe/test_trtllm_gen_fused_moe.py

```
@@ -1975,15 +1980,15 @@ def test_fp8_block_scale_moe_swiglu_oa_activation_param_validation():
     with pytest.raises(ValueError, match="Fp8QuantizationType.MxFp8"):
         trtllm_fp8_block_scale_routed_moe(
             **routed_kwargs,
-            fp8_quantization_type=Fp8QuantizationType.DeepSeekFp8,
+            fp8_quantization_type=Fp8QuantizationType.NoneFp8,
             activation_type=ActivationType.Swiglu.value,
             gemm1_beta=per_expert,
         )
 
     with pytest.raises(ValueError, match="ActivationType.Swiglu"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)